CONTRIBUTING.md

```diff
@@ -1 +1,138 @@
-# Contributing (TODO)
+# Contributing to semantic-release
+
+As a contributor, here are the guidelines we would like you to follow:
+
+- [Code of conduct](#code-of-conduct)
+- [How can I contribute?](#how-can-i-contribute?)
+- [Submitting a Pull Request](#submitting-a-pull-request)
+- [Coding rules](#coding-rules)
+- [Working with the code](#get-started)
+
+We also recommend that you read [How to Contribute to Open Source](https://opensource.guide/how-to-contribute).
+
+## Code of conduct
+
+Please read and follow our [Code of conduct](CODE_OF_CONDUCT.md).
+
+## How can I contribute?
+
+- Improve documentation
+- Give feedback on issues
+- Fix bugs and implement new features
+- [Bug report](https://github.com/teamsempo/SempoBlockchain/issues/new?assignees=&labels=&template=bug_report.md&title=)
+- [Feature request](https://github.com/teamsempo/SempoBlockchain/issues/new?assignees=&labels=&template=feature_request.md&title=)
```

Comment:
Is there some integration between issues and Trello?

Reply:
I did setup a Zapier so that new github issues create a trello card, but not really tbh